package.json

```diff
@@ -92,6 +92,7 @@
     "babel-loader": "8.0.5",
     "babel-plugin-dynamic-import-node": "2.2.0",
     "bulma": "0.7.4",
+    "bulma-divider": "^0.2.0",
```

Comment:
Do you mind changing this to be `0.2.0` (without the `^` at the front) and then re-running `yarn` and committing the results?  Details for the rationale are at https://github.com/JustFixNYC/tenants2/issues/69.

Reply:
✔